h/templates/includes/paginator.html.jinja2

```diff
@@ -1,33 +1,47 @@
 <nav>
-  <ul class="pagination">
+  <ul class="search-results-pager">
```

Comment:
This should probably not say `search-results` if this is to be a reusable component.